plugins/TwoFactorAuth/templates/showRecoveryCodes.twig

```diff
@@ -20,7 +20,7 @@
     <div piwik-content-block
          content-title="{{ 'TwoFactorAuth_TwoFactorAuthentication'|translate }} - {{ 'TwoFactorAuth_RecoveryCodes'|translate }}">
 
-        {% include '@TwoFactorAuth/_showRecoveryCodes.twig' %}
```

Comment:
Isn't that template obsolete now and can be removed?

Reply:
@sgiehl updated.